server/src/test/java/org/elasticsearch/index/analysis/IndexAnalyzersTests.java

```diff
@@ -21,38 +21,13 @@
 
 public class IndexAnalyzersTests extends ESTestCase {
 
-    /**
-     * test the checks in the constructor
-     */
-    public void testAnalyzerMapChecks() {
-        Map<String, NamedAnalyzer> analyzers = new HashMap<>();
-        {
-            NullPointerException ex = expectThrows(
-                NullPointerException.class,
-                () -> new IndexAnalyzers(analyzers, Collections.emptyMap(), Collections.emptyMap())
-            );
-            assertEquals("the default analyzer must be set", ex.getMessage());
```

Comment:
I might have missed this, but has this test moved somewhere?

Reply:
It's already tested implicitly in AnalysisRegistryTests, but I'll add something more explicit.  Basically, if nothing is explicitly configured as `default` then the AnalysisRegistry will always add one when building an `IndexAnalyzers`, so this was testing for something that couldn't happen in production code.